Voice/AI Agent with NodeJS/env.example

```diff
@@ -0,0 +1,8 @@
+# Project ID copied from the API Credentials in your SignalWire Space
+PROJECT_ID=
+# API token copied from the API Credentials in your SignalWire Space
+API_TOKEN=
+# Your SignalWire Space URL copied from the API Credentials in your SignalWire Space
+SPACE_URL=
+# The URL of your deployed application, including the protocol extension (ie https://)
+HOST_APP=
```

Comment:
Could this follow the format defined here? https://github.com/signalwire/examples/wiki/JavaScript-Style-Guide#environment-variables

In particular the format of the comments, with the example line

Reply:
for HOST_APP, we should also specify if the final slash in the url is needed or not